### PR TITLE
Release work

### PR DESCRIPTION
**Pull request by halfak**
Preparing for a PyPI release.  Cleans up readme.rst and adds an example use of Scorer.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -76,6 +72,37 @@ Feature extraction:
...
+    >>> Downloader> d
+    >>> Identifier> wordnet
+    >>> Downloader> d
+    >>> Identifier> omw
+    >>> Downloader> q
+    >>> exit()
+
+
+You might need to install some other dependencies depending on your operating
+system.  These are for ``scipy`` and ``numpy``.
```
Comment (he7d3r) [inline README.rst]: I'm not sure `python-sklearn` is for ``scipy`` and ``numpy`` too.

File: example_scoring.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from mw.api import Session
+
+from revscoring.extractors import APIExtractor
+from revscoring.languages import english
+from revscoring.scorers import MLScorerModel
+
+api_session = Session("https://en.wikipedia.org/w/api.php")
+extractor = APIExtractor(api_session, english)
+
+filename = "models/reverts.halfak_mix.trained.model"
```
Comment (he7d3r) [inline example_scoring.py]: Maybe add a comment mentioning that this file would have been generated by some other command (linking to an example?)

File: README.rst [hunk 3]
```diff
@@ -4,41 +4,37 @@ A generic, machine learning-based revision scoring system designed to be used
...
+        >>>
+        >>> from revscoring.extractors import APIExtractor
+        >>> from revscoring.languages import english
+        >>> from revscoring.scorers import MLScorerModel
+        >>>
+        >>> api_session = Session("https://en.wikipedia.org/w/api.php")
+        Sending requests with default User-Agent.  Set 'user_agent' on api.Session to quiet this message.
+        >>> extractor = APIExtractor(api_session, english)
+        >>>
+        >>> filename = "models/reverts.halfak_mix.trained.model"
```
Comment (he7d3r) [inline README.rst]: Maybe add a comment mentioning that this file would have been generated by some other command (linking to an example?)

File: README.rst (hunk 1 above)
Comment (halfak) [inline README.rst]: Looks like scipy and numpy are the only ones that suffer from those dependencies.  See https://meta.wikimedia.org/wiki/Research_talk:Revision_scoring_as_a_service#Installing_and_checking_requirements.txt

File: example_scoring.py (hunk 2 above)
Comment (halfak) [inline example_scoring.py]: I think that is fair and we should actually have this example added at some point.  Right now, I don't think we have a good place to link to.  And I expect that most of our users won't be building their own models, so this is a more important example to have now.

File: README.rst (hunk 3 above)
Comment (halfak) [inline README.rst]: See note on example_scoring.py

File: README.rst (hunk 1 above)
Comment (he7d3r) [inline README.rst]: I mean, the comment says "g++ git python-sklearn python3.4-dev libatlas-base-dev gfortran" are for `scipy` and `numpy`, but you were able to install them before `scikit-learn`, so `python-sklearn` is not needed for them (but for sklearn itself).

Comment (halfak) [inline README.rst]: Oh!  Yeah, it looks like "python-sklearn" came from one of *your* pull requests.  :P  I'm happy to remove it.

Comment (he7d3r) [inline README.rst]: Ok. I'll let it like this for now and check again later...

**Pull request merged**